### ISSUE TITLE
Added whitespace around tikz picture when svg is made using standalone class with dvisvgm option and a node

### ISSUE THREAD

**Issue by stertooy** [mgieseki/dvisvgm#274]
I'm not certain whether this is a bug with dvisvgm or with the `standalone` class, apologies if this does not belong here. 

When using dvisvgm to make an SVG from the tex-code below, something odd happens: the svg gains a lot of whitespace to the left and to the bottom - as if dvisvgm draws it from the (0,0)-coordinate in TikZ, instead of where the actual image starts.
```
\documentclass[dvisvgm]{standalone}

\usepackage{tikz}
\usepackage[active,tightpage]{preview}
\PreviewEnvironment{tikzpicture}

\begin{document}
	
\begin{tikzpicture}
    \draw (4,4) rectangle (8,6);
    \draw (6,5) node {a};
\end{tikzpicture}

\end{document}
```

> ![tikz-faulty](https://github.com/user-attachments/assets/00b7807f-2fe7-4db8-af2d-d6dbcb9a7d5c)

This does not happen when the `[dvisvgm]` option is omitted from the standalone document class:

> ![tikz-correct](https://github.com/user-attachments/assets/8465dc6c-f190-4238-acd0-b661cd2e7d7f)

Unfortunately, I've also run into instances before where this option is required to create a correct svg, so I can't just omit it entirely. Curiously enough, this only happens when a `node` is present in the TikZ-figure. If I comment out the line with the node, it works correctly whether or not the `dvisvgm`-option is present.

**Comment by mgieseki** [comment]
First of all, there's no need to apologize. Feel free to report any problems regarding dvisvgm here.

But in this case it's actually an issue with TikZ. It's the same as in #272. The letter `a` is positioned with DVI commands but also wrapped by literal SVG elements that shift it to the lower left. Since dvisvgm doesn't evaluate literal SVG fragments embedded in the DVI file, it doesn't know the final position and doesn't adapt the bounding box either. That's why it's too big.

When omitting option `dvisvgm`, TikZ embeds PostScript code which dvisvgm evaluates and therefore allows to compute a thight bounding box.

Long story short, this issue has to be fixed by the TikZ team.

**Comment by Malove98** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.
